submissions/Dovahkiin1991/friends-app/app.js

```diff
@@ -0,0 +1,339 @@
+
+//const ITEMS_PER_PAGE = 10; feature pagination
```

Comment:
Don't leave commented code in the final submission. And an empty first line too.